pinax/documents/views.py

```diff
@@ -273,6 +273,9 @@ class DocumentDelete(LoginRequiredMixin, DeleteView):
     model = Document
     success_url = reverse_lazy("pinax_documents:document_index")
 
+    def get_template_names(self):
```

Comment:
Hi @paltman 

Thanks for the update. Just wanted to clarify your question a bit. I think you're asking if the template name can be specified in the same way that template_name is specified in the DocumentDetail class. The answer is yes!

For some reason I was thinking that the template name _needed_ to be calculated dynamically (for instance showing a different template for post() than for get()). I suppose if we wanted to show a confirmation message that might be a good way to go. But we don't, we want to redirect upon successful deletion. So, specifying the static class variable template_name seems like a no-brainer now. Thanks for the feedback on that. I'll update right away.